airgun/widgets.py

```diff
@@ -1,6 +1,7 @@
 import time
 
 from cached_property import cached_property
+import selenium
```

Comment:
please no direct import, please use the below imports `from widgetastic.exceptions import NoSuchElementException`